code/modules/clothing/under/jobs/Plasmaman/command.dm

```diff
@@ -1,3 +1,11 @@
+/obj/item/clothing/under/plasmaman/adjutant
+	name = "adjutant's plasma envirosuit"
+	desc = "It's a light blue envirosuit, its resistant to the hardest profanities and coffee spills."
+	icon_state = "adjutant_envirosuit"
+	inhand_icon_state = null
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+	armor_type = /datum/armor/plasmaman_captain
```

Comment:
here too

Reply:
fuck, I also forgot to move the adjutant plasmamen suit!